client/functions.lua

```diff
@@ -60,6 +60,33 @@ function QBCore.Functions.DrawText3D(x, y, z, text)
     ClearDrawOrigin()
 end
 
+function QBCore.Functions.CreateBlip(coords, sprite, display, scale, colour, shortRange, title)
+    print(coords, sprite, display, scale, colour, shortRange, title)
+    if not coords or not sprite or not display or not scale or not colour or shortRange == nil or not title then 
+        print("Blip failed to create, most likely missed a setting")
+    else 
+        blip = AddBlipForCoord(coords)
```

Comment:
No consideration for other blip types?

Reply:
Can you elaborate on blip types?
display is the only pram that really changes the type of blip (IE: How the blip is shown), in which it is taken into consideration

Reply:
AddBlipForRadius etc